airflow/models/dag.py

```diff
@@ -1153,6 +1153,7 @@ def clear(
         max_recursion_depth=None,
         dag_bag=None,
         visited_external_tis=None,
+        exclude_task_ids: FrozenSet[str] = frozenset({}),
```

Comment:
This also shouldn't be a frozen set (or even _any_ set) -- but a `typing.Container[str]`